src/plugins/cosmos/components/modals/Send/views/Details.tsx

```diff
@@ -97,6 +104,7 @@ export const Details = () => {
       </ModalHeader>
       <ModalCloseButton borderRadius='full' />
       <ModalBody>
+        <AccountDropdown assetId={(asset as Asset).assetId} onChange={handleAccountChange} />
```

Comment:
no need to cast

Reply:
same comment as regular Send Details - it's form control asset